tools/README.md

```diff
@@ -12,7 +12,41 @@ Typically a SDK is generated by:
 3. Generating SDK source code
 4. Validation and linting for generation process
 
-## Using Tools
+
+## Tools folder structure
+
+- `config` - OpenAPI generator config
+- `releaser` - release automation scripts and documentation
+- `transformer` - OpenAPI transformation engine and it's documentation
+
+## SDK Generation Workflow
+
+```mermaid
```

Comment:
To review: https://github.com/mongodb/atlas-sdk-go/blob/8263cf705dddc4e3cd541ab0978f86f8adefa1a3/tools/README.md